core/app/models/spree/reimbursement_type/reimbursement_helpers.rb

```diff
@@ -37,7 +37,7 @@ def create_refund(reimbursement, payment, amount, simulate)
         reason: Spree::RefundReason.return_processing_reason
       })
 
-      simulate ? refund.readonly! : refund.save!
+      simulate ? refund.readonly! : refund.save! && refund.perform!
```

Comment:
I'd lean towards making this a normal conditional now that the falsy path contains multiple statements. `refund.save!` doesn't ever return false so this code is somewhat misleading looking at a glance.

Reply:
You are right, it's a good chance to improve code here.